lib/ansible/modules/monitoring/icinga2_host.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This module is proudly sponsored by CGI (www.cgi.com) and
+# KPN (www.kpn.com).
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: icinga2_host
+short_description: Manage a host in Icinga2
+description:
+   - "Add or remove a host to Icinga2 through the API"
+   - "( see https://www.icinga.com/docs/icinga2/latest/doc/12-icinga2-api/ )"
+version_added: "2.5"
+author: "Jurgen Brand"
+options:
+  url:
+    description:
+      - HTTP, HTTPS, or FTP URL in the form (http|https|ftp)://[user[:pass]]@host.domain[:port]/path
+    required: true
+  force:
+    description:
+      - If C(yes) and C(dest) is not a directory, will download the file every
```

Comment:
The description looks wrong...

Reply:
I copied the docu from the modules/net_tools/basics/get_url.py module which is also based on the fetch_url function. The first list of parameters (from url to client_key) come from the fectch_url function. https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py

The param is a bit strange for my module but it is there (inherent form fetch_url). So what too do with it ?